script.js

```diff
@@ -240,6 +241,13 @@ document.getElementById('teamTwoColor').oninput = function() {
   renderContent();
 }
 
+document.getElementById('backgroundColor').oninput = function() {
+  if (/^[0-9a-f]{3,6}$/.test(this.value)) {
+    backgroundColor = '#' + this.value;
```

Comment:
Use ES6 string interpolation please!
```suggestion
    backgroundColor = `#${this.value}`;
```